core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2183,6 +2183,13 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Return SparkContext
+   */
+  def get(): SparkContext = {
```

Comment:
@rxin Modified it to private. Any ideas on how to fix this without using SparkContext?